src/tiledb/cloud/file.py

```diff
@@ -26,20 +27,9 @@ def create_file(
     :param async_req: return future instead of results for async support
     :return: FileCreated details, including file_uuid and output_uri
     """
-    warning(
-        DeprecationWarning(
-            "This method will be deprecated from version 0.12.15 onwards. "
-            "Use `tiledb.cloud.files.utils.create_file` instead."
-        ),
-    )
-    utils.create_file(
-        namespace=namespace,
-        input_uri=input_uri,
-        output_uri=output_uri,
-        name=name,
-        access_credentials_name=access_credentials_name,
-        access_credential_name=access_credential_name,
-        async_req=async_req,
+    raise DeprecationWarning(
```

Comment:
Deprecated functions should still work -- otherwise this should be a hard error.